src/main/drivers/accgyro/accgyro_spi_icm456xx.c

```diff
@@ -421,7 +428,8 @@ uint8_t icm456xxSpiDetect(const extDevice_t *dev)
     uint8_t attemptsRemaining = 20;
     uint32_t waited_us = 0;
 
-    spiWriteReg(dev, ICM456XX_REG_BANK_SEL, ICM456XX_BANK_0);
+    // ICM-45686 does not use bank switching (register 0x75 is reserved)
```

Comment:
I am pretty sure this is needed for the 45688. Before we introduced bank switching, we had an erroneous initialization.